src/hdmf/backends/hdf5/h5tools.py

```diff
@@ -41,6 +42,7 @@ def __init__(self, **kwargs):
 
         For `mode`, see `h5py.File <http://docs.h5py.org/en/latest/high/file.html#opening-creating-files>_`.
         '''
+        self.logger = logging.getLogger('%s.%s' % (self.__class__.__module__, self.__class__.__qualname__))
```

Comment:
This seems like a reasonable default logger, but how does a user specify their own logger?